package.json

```diff
@@ -1,25 +1,22 @@
 {
-  "name": "preact-www",
   "private": true,
-  "description": "Preact website.",
+  "author": "Preact Authors <core@preactjs.com>",
+  "license": "MIT",
   "engines": {
-    "node": ">=12"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/preactjs/preact-www.git"
+    "node": ">=18"
   },
   "scripts": {
     "dev": "concurrently \"npm run dev:client\" \"npm run dev:lambda\"",
     "dev:client": "preact watch --no-refresh",
     "dev:lambda": "netlify-lambda serve src/lambda",
     "start": "npm run -s server",
     "prestart": "npm run -s build",
-    "server": "superstatic build -p ${PORT:-8080} --host 0.0.0.0 --gzip -c '{\"rewrites\": [{\"source\":\"**\",\"destination\":\"index.html\"}],\"headers\":[{\"source\":\"**\",\"headers\":[{\"key\":\"Cache-Control\",\"value\":\"max-age=31536000\"}]}]}'",
+    "server": "superstatic build -p ${PORT:-8080} --host 0.0.0.0 --compression -c '{\"headers\":[{\"source\":\"**\",\"headers\":[{\"key\":\"Cache-Control\",\"value\":\"max-age=31536000\"}]}]}'",
```

Comment:
Doesn't look like the rewrite works but we shouldn't want that as we are prerendering all pages.